File: Folder Java/Elektronik.java
public class Elektronik {
    private int id;
    private String nama;
    private String jenis;
    private String lokasi;
    private int rating;
    private String komen;

    public Elektronik(int id, String nama, String jenis, String lokasi, int rating, String komen) {
        this.id = id;
        this.nama = nama;
        this.jenis = jenis;
        this.lokasi = lokasi;
        this.rating = rating;
        this.komen = komen;
    }

    public int getId() { return id; }
    public String getNama() { return nama; }
    public String getJenis() { return jenis; }
    public String getLokasi() { return lokasi; }
    public int getRating() { return rating; }
    public String getKomen() { return komen; }

    public void setNama(String nama) { this.nama = nama; }
    public void setJenis(String jenis) { this.jenis = jenis; }
    public void setLokasi(String lokasi) { this.lokasi = lokasi; }
    public void setRating(int rating) { this.rating = rating; }
    public void setKomen(String komen) { this.komen = komen; }

    public void display() {
        System.out.println("ID: " + id +
                           " | Nama: " + nama +
                           " | Jenis: " + jenis +
                           " | Lokasi: " + lokasi +
                           " | Rating: " + rating +
                           " | Komen: " + komen);
    }
}

File: Folder CPP/Elektronik.cpp
#include <iostream>
#include <string>
using namespace std;

class Elektronik {
private:
    int id;
    string nama;
    string jenis;
    string lokasi;
    int rating;
    string komen;

public:
    Elektronik(int id, string nama, string jenis, string lokasi, int rating, string komen) {
        this->id = id;
        this->nama = nama;
        this->jenis = jenis;
        this->lokasi = lokasi;
        this->rating = rating;
        this->komen = komen;
    }

    int getId() const { return id; }
    string getNama() const { return nama; }
    string getJenis() const { return jenis; }
    string getLokasi() const { return lokasi; }
    int getRating() const { return rating; }
    string getKomen() const { return komen; }

    void setNama(string nama) { this->nama = nama; }
    void setJenis(string jenis) { this->jenis = jenis; }
    void setLokasi(string lokasi) { this->lokasi = lokasi; }
    void setRating(int rating) { this->rating = rating; }
    void setKomen(string komen) { this->komen = komen; }

    void display() {
        cout << "ID: " << id
             << " | Nama: " << nama
             << " | Jenis: " << jenis
             << " | Lokasi: " << lokasi
             << " | Rating: " << rating
             << " | Komen: " << komen << endl;
    }
};
